interfaces/admin_cli.py

```diff
@@ -0,0 +1,501 @@
+        Enter ManageVolunteerMenu for further actions
+        """
+        ManageVolunteerMenu(self.user).cmdloop()
+
+    def do_manage_refugee_profile(self, arg):
+        """
+        Enter ManageCampMenu for further actions
+        """
+        ManageRefugeeMenu(self.user).cmdloop()
+
+    def do_exit(self, arg):
+        super().do_exit(self)
+
+
+class PlanMenu(AdminShell):
+    """
+    PlanMenu is the command line interface for emergency plan management.
+    """
+    plan_options = {"logout": 0, "create_plan": 1, "view_plan": 2,
+                    "close_plan": 3,
+                    "return_main_menu": "r"}
+
+    def plan_menu(self) -> None:
+        print("\033[100m\033[4m\033[1mManage Emergency Plan\033[0m\n"
+              "[ 1 ] Create a new emergency plan\n"
+              "[ 2 ] View an emergency plan\n"
+              "[ 3 ] Close an emergency plan\n\n"
+              "[ R ] Return to Main Menu\n")
+
+    def precmd(self, option: str) -> str:
+        """
+        Transfer option numbers to function name for Admin Menu
+        """
+        if option.isdigit() and int(option) in list(self.plan_options.values()):
+            return list(self.plan_options.keys())[list(self.plan_options.values()).index(int(option))]
+        elif option.upper() == 'X' or option.upper() == 'R':
+            return list(self.plan_options.keys())[list(self.plan_options.values()).index(option.lower())]
+        else:
+            return "invalid_input"
+
+    def preloop(self) -> None:
+        """
+        Display plan menu.
+        """
+        self.plan_menu()
+
+    def return_previous_page(self):
+        while True:
+            option = input("Enter R to return back to the previous menu.\n > ")
+            if option.upper() == 'R':
+                PlanMenu(self.user).cmdloop()
+            else:
+                print("\033[31m {}\033[00m".format("** Invalid input. Enter R to return back to main menu."))
+                continue
+
+    # ----- basic commands -----
+    def do_create_plan(self, arg):
+        """
+        #1 Create a plan
+        """
+        print("\033[100m\033[4m\033[1m{}\033[0m ".format("Create a new emergency plan"))
+        e_name = input("Enter emergency plan name:")
+        # -- handle emergency type
+        print("Enter a emergency type from the list")
+        emergency_types = plan_controller.list_emergency_types()
+        e_types_dict = {t.value: t for t in emergency_types}  # create a dictionary for Enum
+        for key in e_types_dict.keys():
+            print(f'- {key}')
+        while True:
+            e_type = input(">")
+            try:
+                emergency = e_types_dict[e_type]
+                e_type: Plan.EmergencyType = emergency
+                break
+            except KeyError:
+                print(f'Type {e_type} is not on the list. Please re-enter type:')
+
+        e_description = input("Enter description:")
+        e_geo_area = input("Enter geographical area:")
```

Comment:
Very minor point, but if you have a chance could you please add a space after the colon for both e_description and e_geo_area? This will make it a little less cramped when the user types their input in.

Reply:
I've reviewed all again and added a space after all colons. Thanks for the suggestion!